ui/components/ui/export-text-container/export-text-container.component.js

```diff
@@ -5,7 +5,15 @@ import { useI18nContext } from '../../../hooks/useI18nContext';
 import { useCopyToClipboard } from '../../../hooks/useCopyToClipboard';
 import { exportAsFile } from '../../../../shared/modules/export-utils';
 
-function ExportTextContainer({ text = '' }) {
+function ExportTextContainer({
+  text = '',
+  clickHandlerCopy = () => {
+    // default to empty func
+  },
+  clickHandlerDownload = () => {
+    // default to empty func
+  },
```

Comment:
we don't have to default these to any value because on lines 29-31 and 43-45 you check for their existence before calling them. Non-operational methods (noop) should be avoided if possible.

Reply:
Also can we rename these to onClickCopy and onClickDownload?